input/system/tembo/system.go

```diff
@@ -0,0 +1,165 @@
+package tembo
+
+import (
+	"encoding/json"
+	"github.com/pganalyze/collector/config"
+	"github.com/pganalyze/collector/state"
+	"github.com/pganalyze/collector/util"
+	"net/http"
+	"net/url"
+	"strconv"
+)
+
+type Response struct {
+	Status string `json:"status"`
+	Data   Data   `json:"data"`
+}
+
+type Data struct {
+	ResultType string   `json:"resultType"`
+	Result     []Result `json:"result"`
+}
+
+type Result struct {
+	Metric Metric        `json:"metric"`
+	Value  []interface{} `json:"value"`
+}
+
+type Metric struct {
+	Name                  string `json:"__name__"`
+	Endpoint              string `json:"endpoint"`
+	Instance              string `json:"instance"`
+	Job                   string `json:"job"`
+	MetricsPath           string `json:"metrics_path"`
+	Namespace             string `json:"namespace"`
+	Node                  string `json:"node"`
+	PersistentVolumeClaim string `json:"persistentvolumeclaim"`
+	Service               string `json:"service"`
+}
+
+// GetSystemState - Gets system information for a Tembo Cloud instance
+func GetSystemState(config config.ServerConfig, logger *util.Logger) (system state.SystemState) {
+	headers := map[string]string{
+		"Authorization": "Bearer " + config.TemboAPIToken,
+		"accept":        "application/json",
+	}
+
+	client := http.Client{}
+
+	// Get CPU usage percentage
+	query := "sum(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{ namespace=\"" + config.TemboNamespace + "\"}) / sum(kube_pod_container_resource_requests{job=\"kube-state-metrics\",  namespace=\"" + config.TemboNamespace + "\", resource=\"cpu\"})"
+	cpuUsage, err := getFloat64(query, "https://api.data-1.use1.tembo.io/"+config.TemboNamespace+"/metrics/query?query=", client, headers)
+	if err != nil {
+		logger.PrintError("Tembo/System: Encountered error when getting CPU info %v\n", err)
+		return
+	}
+
+	system.CPUStats = make(state.CPUStatisticMap)
+	system.CPUStats["all"] = state.CPUStatistic{
+		DiffedOnInput: true,
+		DiffedValues: &state.DiffedSystemCPUStats{
+			UserPercent: cpuUsage,
+		},
+	}
```

Comment:
@lfittl What should the format of `cpuUsage` look like here? Right now we're receiving something like `0.04305000823670472`. In the pganalyze app it shows a constant 100% utilization.

Reply:
This actually looks correct, we just need to adjust something inside the app to show this correctly based on the system type.